src/main/java/BadApple/main/Ui.java

```diff
@@ -0,0 +1,32 @@
+package BadApple.main;
+
+public class Ui {
+    public static void showWelcome() {
+        String logo = " __      __.__    .__  __             _________                          \n" +
```

Comment:
Assuming the `logo` is a constant name, would it enhance readability by adopting uppercase with underscore notation, e.g. `LOGO`?